db/migrate/20230419084212_add_tree_nodes_overrides.rb

```diff
@@ -25,7 +25,10 @@
         indents.validity_start_date,
         COALESCE(indents.validity_end_date, MIN(replacement_indents.validity_start_date) - INTERVAL '1 second', nomenclatures.validity_end_date) as validity_end_date,
         indents.oid,
-        COALESCE(overrides.depth, indents.number_indents + 2 - (indents.goods_nomenclature_item_id LIKE '%00000000' AND indents.number_indents = 0)::integer) AS "depth"
+        COALESCE(
```

Comment:
Won't this migration have already ran in development, staging, production spaces?

Reply:
Normally yes, but since they'll go out into the same deployment - then no - I'm actually going to cancel CI on the other PR to stop it deploying to staging

Its just avoiding having the two changes as two transactions - I'd split the PRs just for ease of code review but originally they were a single PR